solidity/utils/test_deployment.js

```diff
@@ -290,11 +293,14 @@ const run = async () => {
                 weights
             )
         );
+        console.log('BBB');
```

Comment:
You forgot to remove your debug leftovers.

Reply:
Oops